src/lang-go.ts

```diff
@@ -473,15 +478,15 @@ query FindDependents($query: String!) {
  * - Loop through each repository, create a new connection to the language
  *   server, and call xreferences
  */
-function xrefs({
+async function* xrefs({
     doc,
     pos,
     sendRequest,
 }: {
     doc: sourcegraph.TextDocument
     pos: sourcegraph.Position
     sendRequest: SendRequest<any>
-}): Observable<lspext.Xreference & { currentDocURI: string }> {
+}): AsyncGenerator<lspext.Xreference & { currentDocURI: string }> {
```

Comment:
Return async generator instead of observable so we don't have to go from observable -> async generator -> observable.